packages/auth-compat/test/integration/flows/custom.test.ts

```diff
@@ -137,6 +140,65 @@ describe('Integration test: custom auth', () => {
     expect(customUser!.uid).not.to.eql(anonUser!.uid);
   });
 
+  context('in passthrough mode', () => {
+    beforeEach(async () => {
+      const updatedConfig = await updateEmulatorProjectConfig(
+        JSON.stringify({
+          usageMode: 'PASSTHROUGH'
+        })
+      );
+      expect(updatedConfig).to.eql({
+        signIn: { allowDuplicateEmails: false },
+        usageMode: 'PASSTHROUGH'
+      });
+    });
+
+    afterEach(async () => {
+      await updateEmulatorProjectConfig(
+        JSON.stringify({
+          usageMode: 'DEFAULT'
+        })
+      );
+    });
+
+    xit('signs in with custom token in passthrough mode', async () => {
```

Comment:
Would you mind adding a comment with a TODO(bug#) so we don't forget to enable these tests after launch?